src/components/interaction-designer/block-types/MobilePrimitives_SelectManyResponseBlock.vue

```diff
@@ -1,8 +1,9 @@
 <template>
   <div class="mobile-primitive-select-many-response-block">
     <h3 class="no-room-above">
-      {{ 'flow-builder.edit-block-type' | trans({block_type: trans(`flow-builder.${block.type}`)}) }}
+      {{ `flow-builder.${block.type}` | trans }}
```

Comment:
Thanks, we didn't notice this design change for all blocks. I have to check other blocks.